src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,14 @@
-package core.basesyntax;
+import java.util.Random;
 
 public class ColorSupplier {
+    private Random random = new Random();
+
     public String getRandomColor() {
-        return null;
+        Color[] colors = Color.values();
+        int index = random.nextInt(colors.length);
+        return colors[index].name();
+    }
+    public enum Color {
```

Comment:
The enum Color should have an access modifier to specify its visibility as well. It's also a good practice to place it in a separate file if it's going to be used outside of ColorSupplier.